src/GraphODataTemplateWriter/.config/TemplateWriterSettings.json

```diff
@@ -2,28 +2,30 @@
     "AvailableLanguages": [ "Android", "ObjC", "CSharp", "Python", "JavaScript" ],
     "TargetLanguage": "CSharp",
     "Plugins": [ ],
-    "PrimaryNamespaceName": "com",
+    "PrimaryNamespaceName": "oneDrive",
     "NamespacePrefix": "com",
     "InitializeCollections": "false",
     "AllowShortActions": "false",
-    "NamespaceOverride": "",
+    "NamespaceOverride": "Microsoft.OneDrive.Sdk",
     "TemplatesDirectory": "../../../../Templates",
     "DefaultFileCasing": "UpperCamel",
     "CustomFlags": [ "python2" ],
     "LicenseHeader": [
         "Copyright (c) Microsoft Corporation.  All Rights Reserved.  Licensed under the MIT License.  See License in the project root for license information."
     ],
+    "AdditionalMethodProperties": {
+        "oneDrive.delta": {
+            "CollectionResponse": {
+                "@delta.token": "Edm.String",
```

Comment:
Worth calling out this isn't actually a new hack, just moving it into the config (it's much better in config). The old approach was to have a LongDescription value set to "MethodResponse" on a manually-created entity in the metadata. Then the delta method was updated to return that method response object and convert it to the actual return values for delta.